Norm/Program.cs

```diff
@@ -102,11 +105,19 @@ public static void AddBotServices(HostBuilderContext context, IServiceCollection
                 .Bind(context.Configuration.GetSection(BotOptions.Section))
                 .ValidateDataAnnotations();
 
+            services.AddDbContext<NormDbContext>((servs, opts) =>
+            {
+                var botOptions = servs.GetRequiredService<IOptions<BotOptions>>().Value;
+                var logFactory = servs.GetRequiredService<ILoggerFactory>();

```

Comment:
```suggestion
                BotOptions botOptions = servs.GetRequiredService<IOptions<BotOptions>>().Value;
                ILoggerFactory logFactory = servs.GetRequiredService<ILoggerFactory>();

```

Reply:
I don't use var in this codebase. I prefer explicit types. I know that there is no real difference. It's just the style I prefer. 😅